data/mods/Aftershock/maps/furniture_and_terrain/terrain_spaceship.json

```
@@ -22,6 +22,29 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_afs_augustmoon_diamond_glass",
+    "name": "diamond-matrix glass",
+    "description": "The diamond cubic lattice distorts distant light sources into eight pointed stars.  The galactic core is visible through the glass, but no spacer would ever give it more than a passing glance.",
+    "roof": "t_metal_flat_roof",
+    "symbol": "LINE_XOXO",
+    "color": "light_cyan",
+    "move_cost": 0,
+    "coverage": 100,
+    "flags": [ "TRANSPARENT", "NOITEM", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "OPENCLOSE_INSIDE", "BLOCK_WIND" ]
+  },
+  {
+    "type": "terrain",
+    "id": "t_afs_augustmoon_hull_wall",
+    "name": "Port Augustmoon hull",
+    "description": "The ancient hull of Port Augustmoon, spun into orbit several lifetimes ago.",
+    "roof": "t_metal_flat_roof",
+    "copy-from": "t_wall_metal",
```

```suggestion
    "copy-from": "t_wall_metal",
    "bash": { "str_min": 4000, "str_max": 8000, "sound": "clang!", "sound_fail": "clang!", "ter_set": "t_afs_augustmoon_hull_wall" },
```
Same as before with the diamond glass.